roles/confluent.variables/defaults/main.yml

```diff
@@ -1829,11 +1843,23 @@ ccloud_kafka_broker_enabled: false
 ### Bootstrap Servers to CCloud Kafka
 ccloud_bootstrap_servers: localhost:9092
 
-## CCloud Key
+## CCloud Kafka Key
 ccloud_key: xxxxx
```

Comment:
@rohit2b Question on the inventory variables we want for ccloud connections. As of this commit there are these for kafka:
```
ccloud_kafka_broker_enabled: false
ccloud_bootstrap_servers: localhost:9092
ccloud_key: xxxxx
ccloud_secret: yyyyyy
```
and these for SR:
```
ccloud_schema_registry_enabled: false
ccloud_schema_registry_url: https://localhost:8081
ccloud_schema_registry_key: xxxxx
ccloud_schema_registry_secret: yyyyyy
```

I'm thinking the kafka ones should be `ccloud_kafka_broker_<whatever>` to be more explicit. WDYT?

Reply:
Settled on these with @lokesh-shekar @rohit2b :
```
        ccloud_kafka_enabled: true
        ccloud_kafka_bootstrap_servers: kafka-broker1:9092
        ccloud_kafka_key: client
        ccloud_kafka_secret: client-secret
```